sale_resource_booking/wizards/sale_order_booking_confirm.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Tecnativa - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+from odoo.tests.common import Form
+
+
+class SaleOrderBookingConfirm(models.TransientModel):
+    _name = "sale.order.booking.confirm"
+    _description = "Confirmation dialog to autofill resource bookings"
+
+    order_id = fields.Many2one(
+        "sale.order",
+        string="Order",
+        index=True,
+        required=True,
+        readonly=True,
+        ondelete="cascade",
+    )
+    resource_booking_ids = fields.One2many(
+        related="order_id.order_line.resource_booking_ids", readonly=False
+    )
+
+    def action_invite(self):
+        """Invite booking requesters."""
+        for booking in self.resource_booking_ids:
+            share_f = Form(
+                self.env["portal.share"].with_context(
+                    active_id=booking.id,
+                    active_ids=booking.ids,
+                    active_model="resource.booking",
+                    default_note=booking.requester_advice,
+                    default_partner_ids=[(4, booking.partner_id.id, 0)],
+                )
+            )
+            share = share_f.save()
+            share.action_send_mail()
+
+    def action_noop(self):
+        # At this point, the record was already created with the required
+        # changes; nothing left to do
+        return
```

Comment:
When I confirm a sale order, the record is not reloaded or refreshed, so the user might feel that the order wasn't confirmed.
I think it's necessary to add an action to force the record to refresh.
Please try something like:
```suggestion
        return {"type": "ir.actions.act_window_close"}
```

Reply:
Ok, thanks, actually the right thing to do (in the other method as well) is to use `return {"type": "ir.actions.client", "tag": "reload"}` so that it actually reload and you see the change that the order is confirmed.

Reply:
I suggested using `{"type": "ir.actions.act_window_close"}`, which is used when a wizard is closed and refreshes the current record.
I noticed that the `reload` tag reloads the whole window and causes the breadcrumbs to be lost, but anyway, LGTM.

Reply:
Previously I had used `{"type": "ir.actions.act_window_close"}` but it does not work properly, the same happened in the  `action_invite()` method.

Reply:
What about the following: on module `web_refresher`, create a client action with tag `soft_reload` or similar. On this module, make it depend on it, and call that action?

Reply:
IMO, I think adding web_refresher as a dependency is not a good idea, basically because the use case (refresh) is already covered, and we usually don't like it when new dependencies are added to modules.